Saturday August 31st 2013 4:22 am -04:00,not sure about the videos. but there is this tutorials here on the python official website. if you insist on videos tell me http://docs.python.org/3/tutorial/,None,0
Saturday August 31st 2013 4:20 am -04:00,in the video lectures section and next to each video youll find a mark that means download just hover over it to be sure and it will download,None,0
Saturday August 31st 2013 4:18 am -04:00,activation to python? no you dont need to activate python because its freeif you mean activate win8 yes for sure. ,None,0
Saturday August 31st 2013 4:15 am -04:00,just focus well while you understand the meaning of each choice and check if its really wrong or right, Unresolved,0
Saturday August 31st 2013 4:13 am -04:00,change from flash to html5 or from html5 to flash. one of them should work. :) youll find it at the top of lectures , Unresolved,0
Saturday August 31st 2013 4:09 am -04:00,youre right I thinkbut that wont differ much as number includes float. its not very specified but it works , Unresolved,0
Saturday August 31st 2013 4:04 am -04:00,:) it has no return note that you dont have a return statement in the code and why doesnt count_vowels work with you?, Unresolved,0
Saturday August 31st 2013 3:27 am -04:00,the number of memory addresses are infinite. but the memory addresses that can be used ad the same time are not (depends on your memory).-----------EXx=4x=5x=8if you visualize this code in (visualizer) youll find this happening. line 1 excites. and create a memory address x1 (for example) and put the value 4 in it. then it assigns  memory address x1 to the variable x.line 2 executes. and create a memory address x2 and put the value 5 in it. then it assigns memory address x2 to the variable x. if you note what happens. x1 is no more needed. so its just deleted (Im not sure about the technical term). and only x2 exists.same happens after line3. and this means you only have 1 memory piece busynow check this code.  x=4c=5v=8youll have memory address x1 (value 4) assigned to variable x. and memory address x2 (value 5) assigned to variable c. and memory address x3 (value 8) assigned to variable 8. here there are three pieces of memory are busy at the same time. so the number of memory addresses that can be used at the same time is not infinite but it depends on your memory. , Unresolved,0
Saturday August 31st 2013 3:03 am -04:00,OK first youre using a test version. and I the version were using is Python 3.3.2. also. to see the pop up window you have to be on shell.but first fix the version problem and tell me what happens. , Unresolved,0
Saturday August 31st 2013 3:57 am -04:00,Im not sure what is the visual studio debug is about? and if the problem continues Im not sure why. when I had it I just tried different functions and it did pop up. but still it doesnt pop up with all functions. try different functions. btw abs runs well with me and sorry for late answer, Unresolved,0
Saturday August 31st 2013 4:58 am -04:00,thats because its not in the definition :) this happens I think when the functions parameters doesnt matter. of when the function takes all parameter types and the return will depend on that. if you try this>>>max(fn)nyoull get the result n as shown :) and that just means it also can make a comparison between strings.about abs this is what I get>>> help(abs)Help on built-in function abs in module builtins:abs(...)    abs(number) -> number        Return the absolute value of the argument.and that means its shown in this line specifically abs(number) -> numbertakes a number and returns a number (number means both float and integer) I didnt save definition for area unfortunately so I cant help with it , Unresolved,0
Saturday August 31st 2013 3:11 pm -04:00,I did the same :), Unresolved,0
Saturday August 31st 2013 2:53 am -04:00,I just copied it and ran it into idle and I dont get an error. , Resolved,0
Saturday August 31st 2013 3:10 am -04:00,this is pictures of the problem and its 2 solutions. (explained in my other comment)first: this is the problem he got.second: this is how you solve it in shell.third: this is how you solve it using new window:) enjoyand tell my if the problem continues , Resolved,1
Saturday August 31st 2013 3:00 am -04:00,Oh. I got what went wrong. :D when you want to define a function you always do that in a new window (saved file) not in the shell itself. (that was said in lectures).but if you try it in the shell you have to define the function in a prompt and call it in another prompt. you cant do both in the same prompt (if you dont know prompt means those >>>)so to avoid the error do this. first define the function using this part def even_bigger(x):    return (2 * x) ** xthen call it using this parteven_bigger(12)that way it will work. but as I said its better to do it in a new window.just note that its a syntax error as well. , Resolved,0
Saturday August 31st 2013 3:59 am -04:00,no :) this was a way for you to check your answer. :) but as you submitted and gotten a full mark (congrats :p)  you dont need to resubmit , Resolved,0
